apps/asap-server/test/controllers/research-outputs.test.ts

```diff
@@ -382,6 +383,18 @@ describe('ResearchOutputs controller', () => {
       expect(result.authors).toEqual(expectedAuthorsResponse);
     });
 
+    test('Should return a list of PM emails', async () => {
```

Comment:
testing coverage may be ok but i think there is a few cases we might want to add, for instance:

1) A research output belongs to multiple teams, some of which have a PM
2) A research output belongs to a team with multiple PMs
3) A research output belongs to teams which have the same PMs - the PM email should be deduplicated

I don't know which of these your test fixture covers but personally i like the tests to be explicit, so i would still add all three with a specific test setup and then have the fixture and expectations updated only for some other generic tests

Reply:
1. This is kind of difficult to test explicitly because it's implicit in the fixtures
2. Again, a bit tricky to write a test for this because we just get back an array of emails, difficult to break it down further. But I have renamed one of the emails `multiple-pms-on-same-team@example.com` to make it clearer
3. Have added a test for deduplication

Reply:
yeah but you are not limited to the fixtures right, you can use the fixture but then update it specifically for the test case. Im sure its fairly easy to test, you just update the squidex response types (the ones in fixtures) so that they reflect the specific secnario no?